homeassistant/components/workday/binary_sensor.py

```diff
@@ -43,7 +43,6 @@
 def valid_country(value: Any) -> str:
     """Validate that the given country is supported."""
     value = cv.string(value)
-    all_supported_countries = holidays.list_supported_countries()
```

Comment:
I'm not sure why you opted for calling `list_supported_countries()` in two places rather than the local variable that was there before.

Won't that increase execution time?

It looks like that function is doing a lot of iteration.

https://github.com/dr-prodigy/python-holidays/blob/4a6ae22c473cc5238572ca001c1603e30b3fadc7/holidays/utils.py#L276

Reply:
Sorry, you are right, it's two scopes. (GitHub collapsed it down and I didn't spot it was a completely new function).

Ok to call it twice.

Reply:
No worries, thanks for reviewing this!